src/h2load.cc

```diff
@@ -821,6 +823,22 @@ void Client::on_stream_close(int32_t stream_id, bool success, bool final) {
     }
     ++worker->stats.req_done;
     ++req_done;
+
+    if (worker->log) {
+      auto to_us = [](auto d) {
+        return std::chrono::duration_cast<std::chrono::microseconds>(d);
+      };
+      auto start = to_us(req_stat->request_wall_time.time_since_epoch());
+      auto delta = to_us(req_stat->stream_close_time - req_stat->request_time);
+      *worker->log << start.count() << '\t' << (success ? req_stat->status : -1)
+                   << '\t' << delta.count() << '\n';
+      // Flushing manually is important to ensure atomicity of lines, but
+      // doing it after each line (e.g. with std::endl) is noticeably slow.
+      if (++worker->log_pending == 16) {
+        worker->log->flush();
+        worker->log_pending = 0;
+      }
```

Comment:
I'm wondering write(2) might help atomicity and performance.  C++ stream is a bit slow.

Reply:
Using `write` here would also require something like `snprintf` to format the entry, though, which iirc is actually slower than iostreams due to the overhead of parsing the format string.